pkg/ovs/mcp/mcp.go

```diff
@@ -0,0 +1,288 @@
+package mcp
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
+
+	"github.com/modelcontextprotocol/go-sdk/mcp"
+	"github.com/ovn-kubernetes/ovn-kubernetes-mcp/pkg/kubernetes/client"
+	kubernetesmcp "github.com/ovn-kubernetes/ovn-kubernetes-mcp/pkg/kubernetes/mcp"
+	"github.com/ovn-kubernetes/ovn-kubernetes-mcp/pkg/ovs/types"
+)
+
+// MCPServer provides OVS layer analysis tools
+type MCPServer struct {
+	clientSet *client.OVNKMCPServerClientSet
+}
+
+// NewMCPServer creates a new OVS MCP server
+func NewMCPServer(cfg kubernetesmcp.Config) (*MCPServer, error) {
+	var config *rest.Config
+	var err error
+	if cfg.Kubeconfig != "" {
+		config, err = clientcmd.BuildConfigFromFlags("", cfg.Kubeconfig)
+		if err != nil {
+			return nil, err
+		}
+	} else {
+		config, err = rest.InClusterConfig()
+		if err != nil {
+			return nil, err
+		}
+	}
+
+	clientSet, err := client.NewOVNKMCPServerClientSet(config)
+	if err != nil {
+		return nil, err
+	}
+
+	return &MCPServer{
+		clientSet: clientSet,
+	}, nil
+}
+
+// AddTools registers OVS tools with the MCP server
+func (s *MCPServer) AddTools(server *mcp.Server) {
```

Comment:
There are other tools also mentioned in the [OKEP](https://ovn-kubernetes.io/okeps/okep-5494-ovn-kubernetes-mcp-server/#example-workflow-for-an-end-user) for the OVS layer. I think those are also needed to be added as tools.

Reply:
yes, right, let's do it separately for `retis`.